### ISSUE TITLE
Geht es hier noch weiter?

### ISSUE THREAD

**Issue by Mululu** [J0hnHawk/FS19_Webstats#62]
Hallo, wird hier noch weiter dran gearbeitet?

Vielen Dank!

**Comment by rupertd** [comment]
No, I don't think so. I added a ticket 15 days ago and got no response.  Best option is try and figure out your problem on your own.  It's a great tool, but no focus on it.

**Comment by chris1971nrw** [comment]
Ich versuch auch gerade das Ding zum laufen zu bekommen, vielleicht sollen wir mal den Programmierer fragen ob wir das weiter entwickeln dürfen mit seinem Quell Code

**Comment by tuvok81** [comment]
Ich würde das sehr begrüßen @chris1971nrw  wenn es irgendwie weiter gehen würde, die bedienung ist auch nicht gerade schön :) 
Mir fehlt auch die Alpine Karte

**Comment by J0hnHawk** [comment]
Meinen Segen habt ihr. Gemäß der Lizenz unter der ich meine Arbeit veröffentlicht habt, dürft ihr das Programm verändern und weiterverbreiten. Siehe Punkt 5 der Lizenz. Wär' nur nett, wenn ich als ursprünglicher Author erwähnt bleibe ;-) 

Es gibt/gab auch schon Weiterentwicklungen. Schaut mal unter Forks https://github.com/J0hnHawk/FS19_Webstats/network/members

Ansonsten - ich spiele schon lange kein FS19 mehr - und mich ziehen derzeit auch keine 10 Pferde(stärken) zurück. :-D

**Comment by chris1971nrw** [comment]
wer Lust hat ich habe ein Fork angelegt und werde in den nächsten Tagen auch anfangen was zu tun.
Es würde mich freuen wenn der eine oder andere da mit macht.
https://github.com/chris1971nrw/FS19_Webstats
